### PR TITLE
Updating changelog and readme

### LINKED ISSUE #773: Update changelog and readme
**Describe the feature you'd like**
Add the new releases (6.2.1.0 and 6.2.0.1) to the changelog and update the feature/limitation list in README.md.

### PR DESCRIPTION
**Pull request by sirpalee**
**Changes proposed in this pull request**
- Updating changelog and readme.

**Issues fixed in this pull request**
Fixes #773

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,76 @@
...
+- **Crease Sets**: The procedural now supports crease sets on polymesh. (#694)
+- **Purpose**: Usd Purpose is now supported in the procedural. (#698)
+- **Transform2D**: The procedural now supports remapping UsdTransform2D to built-in Arnold nodes. (#517)
+- **Multi-Threading**: The procedural now uses USD's WorkDispatcher which improves the performance of multi-threaded expansion in many cases. (#690)
+
+#### Render Delegate
+- **Half and Double precision**: Storing data using half or double precision is now supported. (#669)
+- **Light and Shadow linking**: The render delegate now supports light and shadow linking. (#412)
+- **Motion blur for the Point Instancer**: The render delegate now calculates motion blur when using the point instancer. (#653)
+- **Half and Double precision**: Storing data using half or double precision is now supported. (#669)
```
Comment (kikou) [inline CHANGELOG.md:23]: This line is repeated

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,76 @@
...
+- **Pause and Resume**: Pausing and resuming renders are now supported in the render delegate. (#595)
+
+#### Scene Format
+- **Write with default values**: The scene format now supports optionally writing parameters with default values. (#720)
+
+#### Schemas
+- **Removal of the ArnoldUSD DSO**: The Schemas now work without generating a c++ library. This simplifies the build process and removes the need of installing DSOs that are not used. (#705)
```
Comment (kikou) [inline CHANGELOG.md:30]: Incorrect case: C++

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,76 @@
...
+
+#### Render Delegate
+- **Half and Double precision**: Storing data using half or double precision is now supported. (#669)
+- **Light and Shadow linking**: The render delegate now supports light and shadow linking. (#412)
+- **Motion blur for the Point Instancer**: The render delegate now calculates motion blur when using the point instancer. (#653)
+- **Half and Double precision**: Storing data using half or double precision is now supported. (#669)
+- **Pause and Resume**: Pausing and resuming renders are now supported in the render delegate. (#595)
+
+#### Scene Format
+- **Write with default values**: The scene format now supports optionally writing parameters with default values. (#720)
```
Comment (kikou) [inline CHANGELOG.md:27]: The `scope` and `all_atributes` keywords introduced in #721 should be documented somewhere, possibly as Doxygen comments, and mentioned in the release notes. CC @sebastienblor.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,76 @@
...
+
+#### Schemas
+- **Removal of the ArnoldUSD DSO**: The Schemas now work without generating a c++ library. This simplifies the build process and removes the need of installing DSOs that are not used. (#705)
+
+### Bugfixes
+- #715 Initialization order issue with constant strings common source file
+
+#### Build
+- #656 Arnold-usd fails to build with USD 21.02
```
Comment (kikou) [inline CHANGELOG.md:36]: Case: Arnold-USD

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,76 @@
...
+- #663 render_delegate/lihts.cpp fails to compile for pre-21.02 on windows
```
Comment (kikou) [inline CHANGELOG.md:37]: Typo: `lights.cpp`

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,76 @@
...
+- #692 The render delegate fails to build on windows due to template parameter names
```
Comment (kikou) [inline CHANGELOG.md:38]: Case: Windows

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,76 @@
...
+- #687 Crash with empty primvar arrays
+- #679 Attribute subdiv_type should have priority over usd subdivisionScheme
+- #282 Primvars are not inherited from ancestor primitives
+- #215 Issue with instanced primitives' visibility
+- #244 Curves with vertex interpolation on width
+- #732 Support wrap, bias and scale in USdUvTexture
+- #724 ID not passed to the shapes generated in the procedural
+
+#### Render Delegate
+- #651 Error rendering usd file with samples in productName
```
Comment (kikou) [inline CHANGELOG.md:58]: Case: USD

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,76 @@
...
+- #660 Crease Sets and Subdivision scheme is not imported correctly
```
Comment (kikou) [inline CHANGELOG.md:59]: Case: Crease sets and subdivision

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,76 @@
...
+- #727 Arnold does not use wrapS and wrapT values on UsdUVTexture shader node when rendering UsdPreviewSurface
+- #759 Primvars are not correctly set on the instancer if there is more than one prototype
+
+#### Scene Format
+- #615 Usd Writer crashes when node name contains hyphen character
```
Comment (kikou) [inline CHANGELOG.md:64]: Case: USD Writer

**Review by kikou — CHANGES_REQUESTED** [review]
Some minor corrections and remarks to address.

File: CHANGELOG.md (hunk 3 above)
Comment (sirpalee) [reply CHANGELOG.md:26]: Created #789.

**Comment by sirpalee** [comment]
@kikou thank you for the review! Pushed back a new commit with the fixes and created a separate ticket to document the procedural's parameters.

**Comment by kikou** [comment]
I think it would be good to specify the release date for each version as well. Otherwise LTGM.

**Comment by sirpalee** [comment]
@kikou, thanks, good idea! Added the release notes in the same format USD has them in its changelog.

**Pull request merged**